twitchio/message.py

```diff
@@ -61,6 +64,15 @@ def __init__(self, **kwargs):
         self._tags = kwargs.get("tags")
         self.echo = kwargs.get("echo", False)
 
+        if self._tags is not None:
+            first = self._tags.get("first-msg")
+            if first == "1":
+                self.first = True
+            else:
+                self.first = False
+        else:
```

Comment:
You could probably make this even tidier by removing the else clauses and setting a this to False by default

```
        self._tags = kwargs.get("tags")
        self.echo = kwargs.get("echo", False)
        self.first = False

        if self._tags is not None:
            first = self._tags.get("first-msg")
            if first == "1":
                self.first = True
```